float32[] frontier_x # The x-coordinates of the found frontiers (in meter) relative to the map coordinates
float32[] frontier_y # The y-coordinates of the found frontiers (in meter) relative to the map coordinates
float32[] frontier_radius # The radii (in meter) of the circles that enclose the found frontiers
#Note that frontier_x and frontier_y coordinates are the centers of these circles.